frontend/src/modules/settings/containers/DisplayTab.tsx

```diff
@@ -197,11 +257,23 @@ export const DisplayTab = ({ onSettingChange }: Props): JSX.Element => {
     }
     setMonth(change);
   };
+
+  /**
+   * function for handling buttom css based on AM and PM period.
```

Comment:
What is this function needed for? i.e. What changes about button appearance when this function is called?

Reply:
AM/PM is 2 individual button acting in UI as toggle.
So if user clicks on `AM`, it should highlight the button `AM` and dehighlight the button `PM` (Vice versa if user click on `PM` button). This function handles the CSS for the buttons